docs/DeveloperGuide.md

```diff
@@ -856,3 +856,15 @@ testers are expected to do more *exploratory* testing.
        e.g. add a non-alphanumeric character to one of the student's name.<br>
        Expected: UGTeach should discard all data in the file and start with an empty `ugteach.json` file.
 
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Planned Enhancements**
+
+**_Team size: 5_**
+
+1. **Improve duplicate detection to meet real-world use cases**: 
+* The current version only considers students with **both** the same name and the same phone number duplicates. We take the 2 fields as the main criteria for differentiating students, since different students may have the exact same name. However, the email address might also a unique identity for the students. <br>
```

Comment:
Perhaps,

The current version only considers students with **both** the same name and the same phone number as duplicates. We take the 2 fields as the main criteria for differentiating students, since different students may have the exact same name. However, the email address might also be a unique identity for the students.

Reply:
you mean i need to add in "as" right?

Reply:
yeah sorry I rewrote the whole thing HAHAHAH

another one is the last sentence, email address might also "be" a